scripts/ci/diff.sh

```diff
@@ -86,18 +86,150 @@ build_breakdown_cmd () {
 }
 
 build_output_cmd () {
-  output_cmd="${INFRACOST_BINARY} output --no-color --format diff --path $1"
+  output_cmd="$INFRACOST_BINARY output --no-color --format diff --path $1"
   if [ -n "$show_skipped" ]; then
     # The "=" is important as otherwise the value of the flag is ignored by the CLI
     output_cmd="$output_cmd --show-skipped=$show_skipped"
   fi
   echo "${output_cmd}"
 }
 
+MSG_START="💰 Infracost estimate:"
+build_msg () {
+  local include_html=$1
+  local update_msg=$2
+
+  local percent_display
+  local change_word
+  local change_emoji
+  local msg
+
+  percent_display=$(percent_display "$past_total_monthly_cost" "$total_monthly_cost")
+  change_word=$(change_word "$past_total_monthly_cost" "$total_monthly_cost")
+  change_emoji=$(change_emoji "$past_total_monthly_cost" "$total_monthly_cost")
+
+  msg="$MSG_START **monthly cost will $change_word by $(format_cost "$diff_total_monthly_cost")$percent_display** $change_emoji\n"
+  msg+="\n"
+
+  if [ "$include_html" = true ]; then
+    msg+="<table>\n"
```

Comment:
Just curious: Any reason you did this with html and not markdown tables `| a | b | c |`?

Reply:
The only reason was @alikhajeh1 suggested not having bold table headers so it wouldn't detract from the bold "monthly cost will increase..." message. Markdown tables always render the first row cells as `<th>` so they are always bold.